assets/cubyz/biomes/wetlands/base.zig.zon

```diff
@@ -0,0 +1,42 @@
+.{
+	.properties = .{
+		.wet,
+		.hot,
+		.overgrown,
+	},
+	.minHeight = -4,
+	.maxHeight = 4,
```

Comment:
If you want a more consistent water level, it might be best to make min and maxHeight the same.

Reply:
I mean consistent as in each instance of the biome will look similar, like you won't have one instance which is completely submerged.
There will still be a variation in height thanks to the noise map parameters.